File: SWB4/Client/Microsoft Office/INFOTEC WebBuilder 4/WBOffice4/Interfaces/Value.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WBOffice4.Interfaces
{
   public class Value {
    public String key;
    public String title;

    
    public override bool Equals(Object obj)
    {
        if (obj == null)
        {
            return false;
        }
        if (GetType() != obj.GetType())
        {
            return false;
        }
        Value other = (Value) obj;
        if ((this.key == null) ? (other.key != null) : !this.key.Equals(other.key))
        {
            return false;
        }
        return true;
    }

    
    public override int GetHashCode()
    {
        int hash = 7;
        hash = 67 * hash + (this.key != null ? this.key.GetHashCode() : 0);
        return hash;
    }

    
    public override String ToString()
    {
        return title;
    }

}
}

File: SWB4/Client/OpenOffice/OfficeComunicationInterfaces/src/org/semanticwb/office/interfaces/Value.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.semanticwb.office.interfaces;

/**
 *
 * @author victor.lorenzana
 */
public class Value {
    public String key;
    public String title;

    @Override
    public boolean equals(Object obj)
    {
        if (obj == null)
        {
            return false;
        }
        if (getClass() != obj.getClass())
        {
            return false;
        }
        final Value other = (Value) obj;
        if ((this.key == null) ? (other.key != null) : !this.key.equals(other.key))
        {
            return false;
        }
        return true;
    }

    @Override
    public int hashCode()
    {
        int hash = 7;
        hash = 67 * hash + (this.key != null ? this.key.hashCode() : 0);
        return hash;
    }

    @Override
    public String toString()
    {
        return title;
    }

}
